app/uploaders/image_uploader.rb

```diff
@@ -0,0 +1,47 @@
+class ImageUploader < CarrierWave::Uploader::Base
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
```

Comment:
Right now this may not be necessary but in the future you will probably need to move this to the Carrierwave initializer, taking care of choosing `storage :file` if the RAILS_ENV is development or testing, or `storage :fog` if the RAILS_ENV is staging or production